crates/telio-nurse/src/config.rs

```diff
@@ -146,3 +151,23 @@ impl Default for QoSConfig {
         }
     }
 }
+
+/// Configuration for Aggregator component from Nurse
+#[derive(Clone, Default)]
+pub struct AggregatorConfig {
+    /// Collect relay events
+    pub relay_events: bool,
+
+    /// Collect nat traversal events
+    pub nat_traversal_events: bool,
+}
+
+impl AggregatorConfig {
+    /// Create a new Aggregator config
+    pub fn new(features: &FeatureNurse) -> Self {
+        Self {
+            relay_events: features.enable_relay_conn_data.unwrap_or(false),
+            nat_traversal_events: features.enable_nat_traversal_conn_data.unwrap_or(false),
+        }
+    }
+}
```

Comment:
Well, with the current design Nurse doesn't own Aggregator, so if you insist on adding a separate struct for aggregator config, I guess it would be better in the aggregator.rs

Reply:
I have added in this way so that there is less divergence from https://github.com/NordSecurity/libtelio/pull/396 where this is also being added.